### ISSUE TITLE
Path contains regex gives error "contains invalid characters"

### ISSUE THREAD

**Issue by zaniar** [apioo/fusio#63]
I put `/news/$year<[0-9]+>` in the path field when I tried to create a new route.

It gives me
```
/path contains invalid characters inside a path segment in <path to my fusio installation>/vendor/psx/schema/src/Validation/Validator.php on line 91
```

**Comment by k42b3** [comment]
Hi, thanks for the info. It looks like the `path` regexp filter is to restrictive. As alternative you could use the route `/news/:year`.

I have just commited a fix to allow those complex url filters https://github.com/apioo/fusio-impl/commit/ac65255b1e32295b4e89b796466912cbd46efab6 for the next release

**Comment by k42b3** [comment]
So this problem is fixed in the 0.9.7 release
